code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -14,18 +14,18 @@ def __init__(self, **kwargs) -> None:
         # Azure Search
         self.AZURE_SEARCH_SERVICE = os.getenv("AZURE_SEARCH_SERVICE", "")
         self.AZURE_SEARCH_INDEX = os.getenv("AZURE_SEARCH_INDEX", "")
-        self.AZURE_SEARCH_USE_SEMANTIC_SEARCH = os.getenv(
-            "AZURE_SEARCH_USE_SEMANTIC_SEARCH", ""
+        self.AZURE_SEARCH_USE_SEMANTIC_SEARCH = (
+            os.getenv("AZURE_SEARCH_USE_SEMANTIC_SEARCH", "False").lower() == "true"
         )
         self.AZURE_SEARCH_SEMANTIC_SEARCH_CONFIG = os.getenv(
-            "AZURE_SEARCH_SEMANTIC_SEARCH_CONFIG", ""
+            "AZURE_SEARCH_SEMANTIC_SEARCH_CONFIG", "default"
         )
         self.AZURE_SEARCH_INDEX_IS_PRECHUNKED = os.getenv(
             "AZURE_SEARCH_INDEX_IS_PRECHUNKED", ""
         )
-        self.AZURE_SEARCH_TOP_K = os.getenv("AZURE_SEARCH_TOP_K", "")
-        self.AZURE_SEARCH_ENABLE_IN_DOMAIN = os.getenv(
-            "AZURE_SEARCH_ENABLE_IN_DOMAIN", ""
+        self.AZURE_SEARCH_TOP_K = os.getenv("AZURE_SEARCH_TOP_K", 5)
```

Comment:
If you look at how these values were manipulated in the old app.py, do you still think that they should be strings?
AZURE_SEARCH_TOP_K was defaulted to an int (in app.py) and that is the only calling reference
AZURE_SEARCH_ENABLE_IN_DOMAIN has some string manipulation in app.py to convert it into a bool

Reply:
It isn't so much about how the values were manipulated, but rather that since env vars are always strings, it feels safer to me to set the default values also to strings, so that you know the type of variable you are dealing with will always be a string. You then manipulate that variable as a string however you like, rather than a situation where a variable can be either a string or int, depending on whether the env var was set, for example.

However, it's not a blocker, and we're simply replicating the old behaviour, so I'm happy to approve